### DISCUSSION TITLE
Github repository listing

### DISCUSSION THREAD

**Discussion by ankushkatiyar** [community/community#22535] [New to GitHub]
<p>Hi,<br>
recently i notice in dashbaord page of github after login, some repositories not listed and when i access it using <a href="https://github.com/settings/repositories" rel="noopener nofollow ugc">https://github.com/settings/repositories</a>, All repositories displaying<br>
Is it due to any update in application or i need to update something ?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>If by “dashboard” you mean the one right at <code>https://github.com/</code>, that one usually shows only repositories you recently interacted with. If you want to see them all, go to your profile and the “repositories” tab there.</p>
